ql/src/java/org/apache/hadoop/hive/ql/parse/ImportSemanticAnalyzer.java

```diff
@@ -505,9 +506,8 @@ private static ImportTableDesc getBaseCreateTableDescFromTable(String dbName,
 
     Task<?> copyTask = null;
     if (replicationSpec.isInReplicationScope()) {
-      boolean isImport = ReplicationSpec.Type.IMPORT.equals(replicationSpec.getReplSpecType());
       copyTask = ReplCopyTask.getLoadCopyTask(replicationSpec, dataPath, destPath, x.getConf(),
-              isAutoPurge, needRecycle, copyToMigratedTxnTable, !isImport);
+              isAutoPurge, needRecycle, copyToMigratedTxnTable, false);
```

Comment:
Why is this set to false? Do we even need this boolean if everyone is setting it to false?

Reply:
Used with true in some cases.